cmd/plan.go

```diff
@@ -75,18 +76,39 @@ func plan(cmd *cobra.Command, args []string) error {
 func planInstall(rv cran.RVersion, exitOnMissing bool) (*cran.PkgNexus, gpsr.InstallPlan, rollback.RollbackPlan) {
 	startTime := time.Now()
 
-	installedPackages := pacman.GetPriorInstalledPackages(fs, cfg.Library)
-	installedPackageNames := extractNamesFromDesc(installedPackages)
-	log.WithField("count", len(installedPackages)).Info("found installed packages")
-	whereInstalledFrom := pacman.GetInstallers(installedPackages)
+	//Check library existence
+	libraryExists, _ := afero.DirExists(fs, cfg.Library)
```

Comment:
always still worth logging the error and not swallowing it. This is a good example where panic works